### PR TITLE
added some words

### PR DESCRIPTION
**Pull request by vajradevam**
2021-12-23

### CODE REVIEW

File: src/data/Hindi.csv [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 Aasmaan,Aakash
+Agar,Yadi
 Ailaan,Ghoshna
+Ajeeb,Nirala
 Ajeeb,Vichitra
+Akal,Buddhi
+Akhbaar,Samacharpatra
 Amir,Dhani
 Amir,Dhanvaan
+Andaja,Anuman
```
Comment (venusuranus) [inline src/data/Hindi.csv:18]: We already have words using 'z' for the Urdu pronunciation. Hence `Andaza`

Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Andaza,Anuman
```

File: src/data/Hindi.csv [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+Asan,Saral
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Aasaan,Saral
```

File: src/data/Hindi.csv [hunk 3]
```diff
@@ -26,13 +41,21 @@ Dil,Hriday
...
+Ehsaan,Kripa
 Ehsaan,Upkaar
+Ehsaas,Bhavna
 Faasla,Antar
+Faayada,Hit
+Faayada,Laabh
 Faisla,Nirnay
+Farmaan,Aadesh
+Farmaan,Raajghoshna
+Farmaish,Ichha
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Farmaish,Iccha
```

File: src/data/Hindi.csv [hunk 4]
```diff
@@ -48,13 +71,22 @@ Hargiz,Kadaapi
 Hifazat,Raksha
 Hissa,Bhaag
 Ijazat,Anumati
+Ijjat,Kirti
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Izzat,Kirti
```

File: src/data/Hindi.csv [hunk 5]
```diff
@@ -48,13 +71,22 @@ Hargiz,Kadaapi
...
+Ijjat,Maan
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Izzat,Maan
```

File: src/data/Hindi.csv [hunk 6]
```diff
@@ -48,13 +71,22 @@ Hargiz,Kadaapi
...
+Ijjat,Yash
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Izzat,Yash
```

File: src/data/Hindi.csv [hunk 7]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
 Mijaaz,Swabhaav
 Mubarak,Abhinandan
 Mubarak,Badhai
+Mulayam,Komal
 Mumkin,Sambhav
 Mushkil,Kathin
 Musibat,Sankat
+Naach,Nritya
+Naacheej,Moolyaheen
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Naacheez,Moolyaheen
```

File: src/data/Hindi.csv [hunk 8]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Naacheej,Tuchchh
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Naacheez,Tuchchh
```

File: src/data/Hindi.csv [hunk 9]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Naata,Mitrata
+Naata,Moh
```
Comment (arnav-db) [inline src/data/Hindi.csv]: Mitrata and Moh don't seem to fit in the literal meaning of Naata.

File: src/data/Hindi.csv [hunk 10]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Naata,Sambandh
+Nadan,Agyan
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Nadan,Agyaat
```

File: src/data/Hindi.csv [hunk 11]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Nafrat,Ghrina
+Nainsaafi,Anyaaya
+Najayaj,Anuchit
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Naajaayaz,Anuchit
```

File: src/data/Hindi.csv [hunk 12]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
 Musibat,Sankat
+Naach,Nritya
+Naacheej,Moolyaheen
+Naacheej,Tuchchh
+Naata,Mitrata
+Naata,Moh
+Naata,Sambandh
+Nadan,Agyan
+Nafrat,Ghrina
+Nainsaafi,Anyaaya
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Naainsaafi,Anyaaya
```

File: src/data/Hindi.csv [hunk 13]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Najayaj,Anuchit
+Najdeek,Sameep
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Nazdeek,Sameep
```

File: src/data/Hindi.csv [hunk 14]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Nakli,Kritrim
+Nalayak,Ayogya
+Nam,Tar
+Namanjur,Varjit
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Naamanzur,Varjit
```

File: src/data/Hindi.csv [hunk 15]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
 Nasseb,Bhagya
+Naubat,Samay
 Naukar,Sevak
+Nayaab,Durlabh
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Naayaab,Durlabh
```

File: src/data/Hindi.csv [hunk 16]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Nazar,Kripa
```
Comment (arnav-db) [inline src/data/Hindi.csv]: Isn't Nazar - Drishti

File: src/data/Hindi.csv [hunk 17]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Nek,Bhagyawaan
```
Comment (arnav-db) [inline src/data/Hindi.csv]: Bhagyawaan is fortunate.

File: src/data/Hindi.csv [hunk 18]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Nek,Punyaatma
+Nigaah,Drishti
+Nishan,Sanket
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Nishan,Chinha
```

File: src/data/Hindi.csv [hunk 19]
```diff
@@ -101,12 +133,46 @@ Maut,Mrytyu
...
+Noor,Prakash
+Paagal,Moorkh
+Paani,Jal
+Paimaan,Pratigya
+Paimaan,Pushtikaran
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
```

File: src/data/Hindi.csv (hunk 18 above)
Comment (vajradevam) [reply src/data/Hindi.csv]: संकेत => चिन्ह, निशान, लक्षण, प्रतीक [paryavachi shabd]

File: src/data/Hindi.csv [hunk 20]
```diff
@@ -119,13 +185,18 @@ Sheesha,Darpan
 Shukriya,Dhanyavaad
 Sirf,Keval
 Sirf,Maatra
+Taaj,Mukut
+Tabdeel,Badalna
```
Comment (arnav-db) [inline src/data/Hindi.csv]:
```suggestion
Tabdeel,Parivartit
```

**Comment by arnav-db** [comment]
Your additions were great! Thanks a lot for contributing.

**Pull request merged**